### ISSUE TITLE
UI scaling problem in 0.7 and 1.0

### ISSUE THREAD

**Issue by JoToMo1993** [minishmaker/randomizer#70]
The ui of 0.7 as well as 1.0 seems not to scale properly.

<img width="575" alt="image" src="https://github.com/minishmaker/randomizer/assets/5766878/a6c10f68-16de-4c3a-90f1-8e30df5bb4c8">

**Comment by JoToMo1993** [comment]
This is just the tab I have seen it the most, but all but the gerneral tab have this problem.

For reference, I'm using a Window 10 (22H2) PC, with an Intel Core i5-11600 and a RTX3070TI on a 4k monitor.
If you need more data, just reach out to me.

Thanks very much for creating this tool.

**Comment by Myth197** [comment]
This is caused by setting your windows display size to above 100%. It can be resolved by lowering the size to 100% and then reloading the randomizer.

To be able to avoid this issue for the randomizer would require a completely new implementation of the UI that would replace the dynamically generated UI currently used.

**Comment by JoToMo1993** [comment]
So is the general tab the only one, not dynamically generated?

**Comment by Myth197** [comment]
Yes, the General and Advanced tabs are the only ones not dynamically generated
